tests/service/data/SyntaxTree/Nullness/IntListOrNullOrNullOrNull.fs

```diff
@@ -0,0 +1 @@
+let x : int list | null | null | null = []
```

Comment:
At some point, I was having this a valid code, yielding a nested WithNull(WithNull(WithNull(...))) chain of SynTypes.

But I think it probably makes sense to restrict it already in the parser and allow just 1 level of Nullability..